src/Compilers/VisualBasic/Portable/Symbols/SymbolExtensions.vb

```diff
@@ -415,5 +415,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Return vbSymbol
         End Function
 
+        <Extension>
+        Friend Function ContainingNonLambdaMember(member As Symbol) As Symbol
+            While member?.Kind = SymbolKind.Method AndAlso DirectCast(member, MethodSymbol).MethodKind = MethodKind.AnonymousFunction
```

Comment:
Is it recommanded to not use ?. in VB compiler code? I don't think I have time to be verify every single IL instruction.